test/paths.go

```diff
@@ -0,0 +1,48 @@
+/*
+Copyright 2022.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package test
+
+import (
+	"bufio"
+	"os"
+	"strings"
+)
+
+// GetRelativeDependencyPath returns the relative path for the specified module name
+//
+// moduleName string - module name or substring
+//
+// return string
+func GetRelativeDependencyPath(moduleName string) string {
+	var CRDPath string
+	readFile, err := os.Open(os.Getenv("GOMOD"))
```

Comment:
I think we could use a relative path here instead of using GOMOD just in case is not defined in the CI environment for some reason. Your thoughts?

Reply:
The problem with the relative path is that is varies due to that for each tests it `chdir` to its directory. For  `metrics` and `tekton` for instance `../go.mod` is enough but for `controllers/release` and `controllers/results` it is `../../go.mod`.

On commit before this  (which I removed), I added a function to check both places (`../`  `../../`) and returns once `go.mod` is found, but thought that _maybe_ by  setting the var it would require less code and would be more clear from where we are getting it.

Each of them has their advantages. I guess checking `../` and `../../` has the advantage that we don't need to set another variable that we might at some point forget why we need it. The other one is just more straight forward to understand.

Reply:
I will add it to the next commit so you can check it out then we decide what is best.

Reply:
The relative path would be taken always relative to this function (`GetRelativeDependencyPath`), not the caller. I don't think you need the new function at all.